Cargo.toml

```diff
@@ -41,6 +41,9 @@ futures = { version = "0.1.14", optional = true }
 serde = { version = "1.0", optional = true }
 serde_derive = { version = "1.0", optional = true }
 
+# enable rudy via -
+rudy = { optional = true, git = "https://github.com/adevore/rudy" } # pending >0.0.1 release
```

Comment:
My intent was to track `rudy` git on the expectation that its public API is unlikely to change between now and the next crate release. That said, the next release should be Soon™, after which I'll happily point to that instead.